src/averageAges.js

```diff
@@ -36,8 +43,20 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
-function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+function calculateWomenAverageAge(people, withChildren = null) {
+  let women = people.filter(person => person.sex === 'f');
+
+  if (withChildren !== null) {
+    women = people
```

Comment:
The second argument 'withChildren' is expected to be a boolean. Using 'null' as default value may lead to incorrect results when it is not provided. Consider using 'false' as default value.